src/backend/test/1_queries.spec.js

```diff
@@ -43,248 +43,343 @@ const asyncHOF = (fn) => {
 // drawCard: drawCard,
 // customHand: customHand
 
-
-describe('Queries unit testing', _ => {
+describe("Queries unit testing", _ => {
   beforeEach(async () => {
-    try{
+    try {
```

Comment:
instead of dropping tables and resetting you can just delete everything from the tables and seed again. 

https://github.com/Vincit/knex-db-manager seems useful for truncating all the tables in one go